caf-audit/README.md

```diff
@@ -143,7 +143,9 @@ A generated client-side library should be referenced in the normal way in the ap
 Configuration required to be supplied via environment variable:  
 - `CAF_ELASTIC_HOST_AND_PORT_VALUES`: A comma separated list of hostnames and ports to use when contacting elasticsearch. eg. localhost:9200,otherHost:9200  
```

Comment:
Not removing this. As this will be used for documentation to a variable used in the line
https://github.com/CAFAudit/audit-service/blob/7762daaf081c523056fafae8aa5dd026ca1e9b14/caf-audit-binding-elasticsearch/src/main/java/com/hpe/caf/auditing/elastic/ElasticAuditConnection.java#L33